docker/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
Use `#!/usr/bin/env bash`.
See https://stackoverflow.com/questions/21612980/why-is-usr-bin-env-bash-superior-to-bin-bash

Reply:
Interesting read! Thanks for the reference!